src/app/reducers/priceChart.js

```diff
@@ -0,0 +1,26 @@
+import { createReducer } from 'redux-act'
+import {
+  priceChartDataLoading,
+  priceChartDataUpdated,
+  priceChartDataError
+} from 'Actions/priceChart'
+import { createUpserter } from 'Utilities/helpers'
+
+const initialState = {
+}
+
+const chartDataInitialState = {
+  symbol: '',
+  priceData: [],
```

Comment:
Since the name of this in the state is `priceChart` some of the words using in naming are redundant. Try these instead:
`priceData` -> `data`
`lastUpdatedPrice` -> `lastUpdated`
`loadingChartData` -> `loading`
`error` -> `error`
`priceChartDataLoading` -> `priceChartLoading`
`priceChartDataUpdated` -> `priceChartUpdated`
`priceChartDataError` -> `priceChartError`